code/game/objects/structures/mirror.dm

```diff
@@ -172,6 +172,25 @@
 				AC.whitelist = race_list
 				ui_users[user] = AC
 			AC.ui_interact(user)
+		if("Voice")
+			var/voice_choice = input(user, "Perhaps...", "Voice effects") as null|anything in list("Comic Sans", "Wingdings", "Swedish", "Chav")
+			var/voice_mutation
+			switch(voice_choice)
+				if("Comic Sans")
+					voice_mutation = COMICBLOCK
+				if("Wingdings")
+					voice_mutation = WINGDINGSBLOCK
+				if("Swedish")
+					voice_mutation = SWEDEBLOCK
+				if("Chav")
+					voice_mutation = CHAVBLOCK
+			if(voice_mutation)

```

Comment:
They should be able to reset their voice by not selecting anything

Reply:
it's fine as it is, imo, it'd be a mess of code to have a "reset" button that acts as you'd expect (what if you start with one of the mutations roundstart? etc)

Reply:
You could just do an `else return` and when they click cancel on the input, it'll execute that (as `voice_mutation` will be `null` if they cancel)

But I see your concern with roundstart mutations,, no problemo